js/theme.js

```diff
@@ -91,7 +94,7 @@ $(document).ready(function () {
     versionOptions.innerHTML = '';
     versionOptions.appendChild(options);
   }
-  versionNode = document.getElementById("toc-version");
+  let versionNode = document.getElementById("toc-version");
```

Comment:
Babel isn’t configured to transpile the code as far as I can see, so I think we should stay clear of ES6 syntax for now.